internal/ocm/internalid_test.go

```diff
@@ -85,14 +92,13 @@ func TestInternalID(t *testing.T) {
 			str := internalID.String()
 			assert.Equal(t, tt.path, str)
 
-			if kind == cmv1.NodePoolKind {
+			if kind == cmv1.NodePoolKind || kind == arohcpv1alpha1.NodePoolKind {
```

Comment:
Don't need to check for both, it's the same string `"NodePool"`.
[cmv1.NodePoolKind](https://pkg.go.dev/github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1#NodePoolKind)
[arohcpv1alpha1.NodePoolKind](https://pkg.go.dev/github.com/openshift-online/ocm-sdk-go/arohcp/v1alpha1#NodePoolKind)